### DISCUSSION TITLE
I need help with Refresh rate

### DISCUSSION THREAD

**Discussion by p4ckett** [hyprwm/Hyprland#3706] [Q&A] [answered]
I have 240hz monitor but xrandr only shows 60HZ n stuff

Screen 0: minimum 16 x 16, current 1920 x 1080, maximum 32767 x 32767
HDMI-A-1 connected 1920x1080+0+0 (normal left inverted right x axis y axis) 540mm x 300mm
   1920x1080     59.96*+
   1440x1080     59.99  
   1400x1050     59.98  
   1280x1024     59.89  
   1280x960      59.94  
   1152x864      59.96  
   1024x768      59.92  
   800x600       59.86  
   640x480       59.38  
   320x240       59.52  
   1680x1050     59.95  
   1440x900      59.89  
   1280x800      59.81  
   1152x720      59.97  
   960x600       59.63  
   928x580       59.88  
   800x500       59.50  
   768x480       59.90  
   720x480       59.71  
   640x400       59.95  
   320x200       58.96  
   1600x900      59.95  
   1368x768      59.88  
   1280x720      59.86  
   1024x576      59.90  
   864x486       59.92  
   720x400       59.55  
   640x350       59.77

**Comment by MightyPlaza** [comment, marked as answer]
xrandr is for X11
in hyprland you can use wlr-randr if you want

like it says in the wiki (https://wiki.hyprland.org/Configuring/Monitors/), you can just use highrr to have the highest refresh rate
`monitor=,highrr,auto,1`

**Reply by p4ckett** [reply]
Thanks for helping, another questions, I cannot screen share.

**Reply by MightyPlaza** [reply]
there are many causes for that
is the hyprland portal installed and running?
nvidia gpu?
https://wiki.hyprland.org/Useful-Utilities/Screen-Sharing/

**Reply by p4ckett** [reply]
I have Nvidia GPU yes, RTX 3060, and also I cannot figure out how to change the refresh rate because I always used xrandr can you give me command?

**Reply by MightyPlaza** [reply]
you have to use hyprland.conf (https://wiki.hyprland.org/Configuring/Monitors/)
one example would be `monitor=DP-1, 1920x1080@240, 0x0, 1`
keep in mind you might need to lower the refresh rate if hyprland doesn't start (because of nvidia)

**Reply by p4ckett** [reply]
Alright where is the hyprland.conf is located?

**Reply by MightyPlaza** [reply]
`~/.config/hypr/hyprland.conf`
all this is explained in the wiki https://wiki.hyprland.org/Configuring/Configuring-Hyprland/

**Reply by p4ckett** [reply]
can you list the stuff I need to do? thanks :^)

**Reply by MightyPlaza** [reply]
simply put `monitor=DP-1, 1920x1080@240, 0x0, 1` in `~/.config/hypr/hyprland.conf` and change the port and resolution to the one you want (you can get the port and current resolution by running hyprctl monitors)

**Comment by p4ckett** [comment]
I configured refresh rate, just need help with the screen share. thanks!

**Reply by MightyPlaza** [reply]
assuming you have xdg-desktop-portal-hyprland installed and running `systemctl --user status xdg-desktop-portal-hyprland`
you need the nvidia patch (`hyprland-nvidia` package from aur)
this should be all that is needed for nvidia afaik, but if it still doesn't work you should try to check the hyprland-nvidia channel on the discord server for current fixes for nvidia
